SensorCode/SensorState.py

```diff
@@ -1,29 +1,38 @@
 from typing import List, Dict
-
+import LIDAR_API
+import TERABEE_API
+import IMU_API
+import TEST_API
 
 class SensorState:
     """
     Class to keep track of the state of the sensor inputs for C1C0, to be stored on Jetson
         Instance Attributes:
             lidar (List[tuple[int, int]]): List of lidar values, each element is an (ang,dist) tuple
             terabee_bot (List[int]): List of bottom terabee distances, each element is distance
+            teerabee_mid (List[int]): List of mid terabee distances, each element is distance
             terabee_top (List[int]): List of top terabee distances, each element is distance
             pos_x (int): X position of hedgehog in GPS sub-map
             pos_y (int): Y position of hedgehog in GPS sub-map
         Class Attributes:
             terabee_bot_ang (Dict[int, int]): mapping from indices in terabee array's to angle of reading
+            terabee_mid_ang (Dict[int, int]): mapping from indices in terabee array's to angle of reading
             terabee_top_ang (Dict[int, int]): mapping from indices in terabee array's to angle of reading
     """
     terabee_bot_ang: Dict[int, int] = {}
+    terabee_mid_ang: Dict[int, int] = {}
     terabee_top_ang: Dict[int, int] = {}
 
     def __init__(self):
         # initialize to max-size values for socket bytesize testing
         self.lidar: List[tuple[int, int]] = [1000]*360
-        self.terabee_bot: List[int] = [1]*20
-        self.terabee_mid: List[int] = [1]*20
-        self.terabee_top: List[int] = [1]*20
+        self.terabee_bot: List[int] = [1]*8
+        self.terabee_mid: List[int] = [1]*8
+        self.terabee_top: List[int] = [1]*8
+        self.imu_gyro: List[int] = [1]*3
+        self.imu_linear_acc: List[int] = [1]*3
         self.heading: int = 0
+        LIDAR_API.init_serial('/dev/ttyTHS1', 38400)
```

Comment:
Maybe add a comment here to indicate that the port names will change depend on the machine, so we can make sure to change it at the right times